api_v3/lib/types/entry/KalturaLiveEntry.php

```diff
@@ -36,10 +36,17 @@
 	/**
 	 * Elapsed recording time (in msec) up to the point where the live stream was last stopped (unpublished).
 	 * @var int
+	 * @deprecated
 	 */
 	public $lastElapsedRecordingTime;
 
 	/**
+	 * Elapsed live streaming time (in msec) up to the point where the live stream was last stopped (unpublished).
+	 * @var int
+	 */
+	public $lastElapsedLiveDuration;
```

Comment:
@AviBueno, are you going to set this attribute from the media-server? Why can't you use the duration as passed to the append recording?